### ISSUE TITLE
DIAG-CMDS fails

### ISSUE THREAD

**Issue by stacksmith** [JadedCtrl/cl-ipfs-api2#1]
The very first thing I tried...

It appears that the actual low-level IPFS-CALL returns the parsed JSON as a bunch of hashtables, but DIAG-CMDS then tries to convert them to conses using RE-HASH-TABLE-ALIST, and fails:
```
The value
  0
is not of type
  LIST
   [Condition of type TYPE-ERROR]

Restarts:
 0: [RETRY] Retry SLIME REPL evaluation request.
 1: [*ABORT] Return to SLIME's top level.
 2: [ABORT] abort thread (#<THREAD "repl-thread" RUNNING {1001570213}>)

Backtrace:
  0: (TEST-APPLY #<FUNCTION HASH-TABLE-P> #<FUNCTION ALEXANDRIA:HASH-TABLE-ALIST> ("ID" . 0))
  1: (TEST-APPLY #<FUNCTION HASH-TABLE-P> #<FUNCTION ALEXANDRIA:HASH-TABLE-ALIST> (("ID" . 0) ("Args" "Qme2sLfe9ZMdiuWsEtajWMDzx6B7VbjzpSC2VWhtB6GoB1/wiki/Peer-to-peer.html") ("Options" . #<HASH-TABLE :TES..
  2: (DIAG-CMDS)
  3: (SB-INT:SIMPLE-EVAL-IN-LEXENV (DIAG-CMDS) #<NULL-LEXENV>)
```

**Comment by stacksmith** [comment]
REHASH-TABLE-ALIST is at fault... As a quick and dirty solution I rewrote it as
```
(defun re-hash-table-alist (hash-table)
  "Turn a hash-table into an associative list, recursively-- if any of the
  hash-table's values are ther hash-tables, they too are turned into alists."
  (loop for k being the hash-keys in hash-table
	using (hash-value v)
	collect (cons k (if (hash-table-p v)
			    (re-hash-table-alist v)
			    v))))
```

**Comment by JadedCtrl** [comment]
Should be fixed with f185501.

**Comment by stacksmith** [comment]
Works as expected!  Thank you.
